packages/civic-sandbox/src/state/sandbox/selectors.js

```diff
@@ -12,53 +12,54 @@ export const getState = createSelector(
 
 export const getSandbox = createSelector(
   rootState,
-  ({ sandbox }) => sandbox
+  ({ sandbox = {} }) => sandbox
 );
 
 const getProperty = key => createSelector(getState, state => state[key]);
 
 const getSandboxProperty = key => createSelector(getSandbox, state => state[key]);
 
-export const isSandboxLoading = getProperty('sandboxPending');
-export const isFoundationLoading = getProperty('foundationPending');
-export const areSlidesLoading = getProperty('slidesPending');
-export const getSandboxError = getProperty('sandboxError');
-export const getSandboxData = getProperty('sandbox');
-export const getFoundations = getProperty('foundations');
-export const getFoundationError = getProperty('foundationError');
-export const getSelectedPackage = getProperty('selectedPackage');
+export const isSandboxLoading = getSandboxProperty('sandboxPending');
```

Comment:
Not that it matters, but `getSandboxProperty` is now doing what `getProperty` was intended to do. /shrug

Reply:
yeah, could definitely use some clean up.